snippets/cpp/VS_Snippets_Remoting/Httpwebrequest_HaveResponse/CPP/httpwebrequest_haveresponse.cpp

```diff
@@ -23,15 +23,15 @@ int main()
 {
    try
    {
-      
+
       // <Snippet1>
       // Create a new 'HttpWebRequest' Object.
       HttpWebRequest^ myHttpWebRequest = dynamic_cast<HttpWebRequest^>(WebRequest::Create( "http://www.contoso.com" ));
       HttpWebResponse^ myHttpWebResponse;
-      
+
       // Display the 'HaveResponse' property of the 'HttpWebRequest' object to the console.
       Console::WriteLine( "\nThe value of 'HaveResponse' property before a response object is obtained : {0}", myHttpWebRequest->HaveResponse );
-      
+
       // Assign the response object of 'HttpWebRequest' to a 'HttpWebResponse' variable.
       myHttpWebResponse = dynamic_cast<HttpWebResponse^>(myHttpWebRequest->GetResponse());
       if ( myHttpWebRequest->HaveResponse )
```

Comment:
Also, it would be great if you can format this C++ file, it looks strange a bit.

Reply:
Seeing that touching this file broke the build, I'll just revert.

Reply:
I meant these:

```suggestion
      if (myHttpWebRequest->HaveResponse)
```

just few whitespaces changes in function calls and statements, but it's not really required, if you want to you can leave it as-is.

Reply:
> Seeing that touching this file broke the build, I'll just revert.

No need to revert it, you can simply ignore this error in cpp files:

`Error: 😵 Project missing. A project (and optionally a solution file) must be in this directory or one of the parent directories to validate and build this code.`